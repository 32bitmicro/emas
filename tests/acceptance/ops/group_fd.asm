AD r0
SD r0
MW r0
DW r0
AF r0
SF r0
MF r0
DF r0
