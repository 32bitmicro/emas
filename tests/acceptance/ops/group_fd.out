@ 0x0000 : 0x7c00  /  011 111 0 000 000 000  /  31744
@ 0x0001 : 0x7c40  /  011 111 0 001 000 000  /  31808
@ 0x0002 : 0x7c80  /  011 111 0 010 000 000  /  31872
@ 0x0003 : 0x7cc0  /  011 111 0 011 000 000  /  31936
@ 0x0004 : 0x7d00  /  011 111 0 100 000 000  /  32000
@ 0x0005 : 0x7d40  /  011 111 0 101 000 000  /  32064
@ 0x0006 : 0x7d80  /  011 111 0 110 000 000  /  32128
@ 0x0007 : 0x7dc0  /  011 111 0 111 000 000  /  32192
